frontend/src/components/OperationSteps/OperationSteps.tsx

```diff
@@ -50,15 +65,27 @@ const OperationSteps: FunctionComponent<OperationStepsProps> = (props) => {
               const isActiveStep = activeStep && activeStep.get('step_id') === step.get('step_id');
 
               return (
-                <StyledListItem
-                  className="py-2"
-                  key={index}
-                  onClick={!activeStep && onClickStep(step)}
-                  disabled={(activeStep && !isActiveStep) || props.disabled}
-                  active={isActiveStep}
-                >
-                  <OperationStep step={step} />
-                </StyledListItem>
+                <StyledStepContainer key={index}>
+                  <StyledListItem
+                    className="py-2"
+                    onClick={!activeStep && onClickStep(step)}
+                    disabled={(activeStep && !isActiveStep) || props.disabled}
+                    active={isActiveStep}
+                  >
+                    <OperationStep step={step} />
+                  </StyledListItem>
+                  <div className="step-action-buttons d-inline-flex">
+                    <StyledButton
+                      title="Duplicate"
+                      variant="link"
+                      size="sm"
+                      data-testid="step-duplicate"
+                      onClick={() => props.onDuplicateStep(step)}
+                    >
+                      <i className="material-icons">content_copy</i>
+                    </StyledButton>
+                  </div>
```

Comment:
Your styling seems to only hide the buttons, but the entire row is always showing, which leaves extra padding when one isn't hovering on the button. Update the styling to show/hide the entire actions row, not just the buttons.


How it is:
![image](https://user-images.githubusercontent.com/5672438/112942776-516a0c80-9139-11eb-90b2-604ef40b1aa7.png)


How it should be:
![image](https://user-images.githubusercontent.com/5672438/112942719-3c8d7900-9139-11eb-99dc-4e3342cb6f73.png)